zio-http/src/main/scala/zio/http/Cookie.scala

```diff
@@ -31,7 +31,9 @@ final case class Cookie[T](name: String, content: String, target: Cookie.Type[T]
    */
   def encode(validate: Boolean)(implicit ev: CookieEncoder[T]): Either[Exception, String] =
     try {
-      Right(ev.unsafeEncode(self, validate))
+      Unsafe.unsafe { implicit u =>
```

Comment:
I think when we are inside this library, we should always pass `Unsafe.unsafe` explicitly or import it at the top of the file. We should aim to have net zero performance impact.